README.md

```diff
@@ -159,6 +185,49 @@ func (r *DeleteAuthorsReq) Query() string {
 }
 ```
 
+The same can be done with `sql` package predefined types
+
+```go
+type DeleteAuthrosReq struct {
+	Tags sql.Int64Array `sql:"tags"`
+}
+
+func (r *DeleteAuthorsReq) Query() string {
+	return `DELETE FROM authors WHERE tags=ANY(@tags::UUID[])`
+}
+```
+
+##  Nested types
+
+```go
+type Author struct {
+    ID   int64  `dql:"id"` 
+    Name string `sql:"name"`
+}
+
+type Book struct {
+    ID   int64  `sql:id`
+    Name string `sql:"name"`
+    Desc string `sql:"description"`
+    Author Author
+}
+type CreateBookReq struct {
+    Book Book
+}
+
+func (r *CreateBookReq) ProcessRow(rowMap sal.RowMap) {
+	rowMap.Set("author_id", r.Book.Author.ID)
+	rowMap.Set("book_id",   r.Book.ID)
+	rowMap.Set("book_name", r.Book.Name)
+	rowMap.Set("book_desc", r.Book.Desc)
+}
+
+func (r *CreateBookReq) Query() string {
+	return `INSERT INTO books (id, author_id, name, description)
+	VALUES (@book_id, @author_id, @book_name, @book_desc)`
```

Comment:
I think it produces missunderstanding: `Desc`, `description`, `@book_desc`

Reply:
check it